### PR TITLE
feat: Authentication Page And Profile Page

### PR DESCRIPTION
**Pull request by vishal804**
Hello 

Created Signin and Signup Functionality 

Created Profile Page And Update Profile 

### Files to Review 

src/component/header/Header.js

src/component/profileEditModal/ProfileEditModal.js
src/component/sideNav/SideNav.js

src/pages/authenticationPage/Signin.js
src/pages/authenticationPage/Signup.js
src/pages/landingPage/LandingPage.js

src/pages/profilePage/ProfilePage.js

src/redux/reducer/authenticationSlice.js

### CODE REVIEW

**Review by manojasarsa — APPROVED** [review]
Code seems clean. GTG

File: src/component/profileEditModal/profileEditModal.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+.editProfile-container {
+  margin: 0 auto;
+  width: 50%;
+  text-align: center;
+  background-color: #f7f7f7;
+}
+
+/* .profile-picture {
```
Comment (MarkVed17) [inline src/component/profileEditModal/profileEditModal.css]: Watch out for unnecessary code.

File: src/pages/authenticationPage/Signup.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    password: "",
+    confirmPassword: "",
+    firstName: "",
+    lastName: "",
+  });
+
+  const sumbitHandler = (e) => {
+    e.preventDefault();
+    if (
+      userCredentials.password.length >= 9 &&
```
Comment (MarkVed17) [inline src/pages/authenticationPage/Signup.js:23]: I liked that you enforced a length constraint.

**Review by MarkVed17 — APPROVED** [review]
You've followed good coding practices. Makes it real quick to understand the code. Good to proceed.

File: src/component/profileEditModal/profileEditModal.css (hunk 1 above)
Comment (vishal804) [reply src/component/profileEditModal/profileEditModal.css]: Done

File: src/pages/authenticationPage/Signup.js (hunk 2 above)
Comment (vishal804) [reply src/pages/authenticationPage/Signup.js:23]: Thank You

**Comment by vishal804** [comment]
> Code seems clean. GTG

Thanks For Review

**Pull request merged**